### PR TITLE
Permission added to REST Server Broadcast .

### PR DESCRIPTION
**Pull request by TheBambino**
Nub trying this again.

**Feature:**
If a REST token is breached, there could be potential spam issues. Where each token perm could pose a potential risk, not having perms on ServerBroadcast makes it an issue if any token is breached. It also gives more control over those that have access to the token.

### CODE REVIEW

File: TShockAPI/Rest/RestPermissions.cs [hunk 1]
```diff
@@ -67,6 +67,9 @@ public static class RestPermissions
 		[Description("REST user can reload configurations, save the world and set auto save settings.")]
 		public const string restcfg = "tshock.rest.cfg";
 
+		[Description("REST user can turn off / restart the server.")]
```
Comment (bartico6) [inline TShockAPI/Rest/RestPermissions.cs]: Wrong description? :p

**Review by bartico6 — CHANGES_REQUESTED** [review]
LGTM once the comments are addressed.

Also, should determine if this was intended behaviour or not.

File: TShockAPI/Rest/RestPermissions.cs (hunk 1 above)
Comment (TheBambino) [reply TShockAPI/Rest/RestPermissions.cs]: Yes, I caught that earlier. Changing it now

**Comment by TheBambino** [comment]
> LGTM once the comments are addressed.

Done.

**Review by bartico6 — APPROVED** [review]
While I can't accept PRs (nor do my reviews mark it as ready for merge), I'll say that everything looks good now

**Review by hakusaro — DISMISSED** [review]
Can you update the changelog?

**Review by QuiCM — APPROVED** [review]

**Pull request merged**